demo/graphql/queries.gql

```diff
@@ -251,3 +251,44 @@ query PkgQ9 {
     ...allPkgTree
   }
 }
+
+fragment allSourceTree on Source {
+  id
+  type
+  namespaces {
+    id
+    namespace
+    names {
+      id
+      name
+      tag
+      commit
+    }
+  }
+}
+
+fragment allCertifyLegalTree on CertifyLegal {
+  id
+  subject {
+    __typename
+    ... on Package {
+      ...allPkgTree
+    }
+    ... on Source {
+      ...allSourceTree
+    }
+  }
+  declaredLicense
+  discoveredLicense
+  attribution
+  justification
+  timeScanned
+  origin
+  collector
+}
+
+query CertifyLegalQ1 {
```

Comment:
Hey @robert-cronin, looks like this specific query only returns the license found in the SPDX document. You can use this one:

```


query CertifyLegal {
    CertifyLegal(certifyLegalSpec: {subject: {package: {name: "log4j-core"} } }) {
        ...AllCertifyLegalTree
    }
}
```

which will return both!

Reply:
Ah that is good to know, thank you for the updated query!